protocols/v2/framing-sv2/src/framing2.rs

```diff
@@ -105,8 +111,7 @@ impl<'a, T: Serialize + GetSize, B: AsMut<[u8]> + AsRef<[u8]>> Frame<'a, T> for
     type Buffer = B;
     type Deserialized = B;
 
-    /// Serialize the frame into dst if the frame is already serialized it just swap dst with
-    /// itself
+    /// Write the serialized `Sv2Frame` into `dst`.
```

Comment:
I would say (also if it seems obvious but can be helpful when you write code not having to check the implementation for things like that) that this operation when called on an already serialized frame is very cheap (consume the Sv2Frame) but when called on a non serialized frame is not so cheap cause it have to serialize it.

Reply:
1fe9e1af00ef2f09a30862b8f4aa60cfc4af6819